.release/release-metadata.hcl

```diff
@@ -3,3 +3,4 @@ url_docker_registry_ecr = "https://gallery.ecr.aws/hashicorp/consul-ecs"
 url_license           = "https://github.com/hashicorp/consul-ecs/blob/main/LICENSE"
 url_project_website = "https://www.consul.io/docs/ecs"
 url_source_repository = "https://github.com/hashicorp/consul-ecs"
+url_release_notes = "https://www.consul.io/docs/release-notes"
```

Comment:
I wasn't able to link directly to the consul-ecs section of the release notes, fwiw